pkgs/games/mari0/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, zip, love_11, lua, makeWrapper, makeDesktopItem }:
+
+let
+  pname = "mari0";
+  version = "1.6.2";
```

Comment:
I think you should move pname and version into the mkDerivation attrset and make the attrset recursive with rec and delete name from that set.

Reply:
So you would also move the `makeDesktopItem` inline then, since it relies on pname?

I've adapted this from https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/rimshot/default.nix

Reply:
Oh I missed that the desktopItem was using pname. In that case, you can use `inherit pname version;` inside the mkDerivation instead of name

Reply:
Cool, thanks!